aws/stage/rds/outputs.tf

```
@@ -0,0 +1,3 @@
+output "db_endpoint" {
+  value = aws_db_instance.postgres.endpoint
```

with address we have the endpoint + port
```suggestion
  value = aws_db_instance.postgres.address
```